src/material-examples/datepicker-views-selection/datepicker-views-selection-example.css

```diff
@@ -1 +1,7 @@
-/** No CSS for this example */
+.example-month-picker .mat-calendar-period-button {
+    pointer-events: none;
```

Comment:
The indentation seems wrong in this file.

Reply:
Fixed